locales/en-US/app.l20n

```diff
@@ -104,8 +103,9 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainer "There are <span data-l10n-id="userCount" class="bold" data-hook="install-count"></span> people trying <span data-hook="title"></span> right now!">
```

Comment:
> GIFs are never stupid comments.

@chuckharmston: Explain that to all the repo owners who keep seeing this GIF I keep recycling today:
![](https://media.giphy.com/media/kxNQqpj1wTiX6/giphy.gif)